redisinsight/api/config/features-config.json

```diff
@@ -47,6 +47,28 @@
         }
       }
     },
+    "documentationChat": {
+      "flag": true,
+      "perc": [[0,100]],
```

Comment:
let's go with [0,25] for both chatbots for now, and gradually roll it out

Reply:
done